src/functions.date.js

```diff
@@ -98,6 +98,16 @@ $.extend(_, {
 
 _.msTo = (what, ms) => Math.floor(Math.abs(ms) / (s[what] * 1000)) || 0;
 
+_.toMs = function (what, value) {
```

Comment:
If I understand what this does correctly, I don't think it's needed. We typically do it via e.g. `5 * days()` in Mavo expressions, which translates to `5 * Mavo.Functions.days()` in JS.